### PR TITLE
initial commit by sachintha

### PR DESCRIPTION
**Pull request by sachinthazoysa**

### CODE REVIEW

**Review by janakachinthana — COMMENTED** [review]
reviewed by janaka

File: e-commerce-app-frontend/index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title>Document</title>
   </head>
   <body>
-    <h1>Hello</h1>
+    <h1>Hello world</h1>
```
Comment (janakachinthana) [inline e-commerce-app-frontend/index.html]: this should be in h2 tag

Comment (sachinthazoysa) [reply e-commerce-app-frontend/index.html]: Resolved

**Review by janakachinthana — COMMENTED** [review]
done

**Pull request merged**